src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
+
+const averageAge = (people) => people.reduce(
+  (sum, person) => sum + person.died - person.born, 0)
+  / people.length;
+
```

Comment:
does `averageAge` name sounds good for a function?

Reply:
Got it, will correct, thanks!